tool/src/main/java/com/fidesmo/fdsm/Main.java

```diff
@@ -272,7 +272,7 @@ public static void main(String[] argv) {
 
                 // Allows to run with any card
                 if (args.has(OPT_QA)) {
-                    String number = Integer.toString(new Random().nextInt(900000) + 100000).substring(0, 6);
+                    String number = Integer.toString(ThreadLocalRandom.current().nextInt(900000) + 100000).substring(0, 6);
```

Comment:
IMO it was better previously :)

Reply:
Did not know about ThreadLocalRandom (as normally only use SecureRandom) but this happens to please spotbugs as well.

Reply:
I mean it's `main` thread so leaving random attached to it makes little sense as it's never re-used. But if it pleases spotbugs that's fine.

Reply:
Focus here is "pleases spotbugs", yes.